src/utilstrencodings.cpp

```diff
@@ -39,6 +39,46 @@ std::string SanitizeString(const std::string& str, int rule)
     return strResult;
 }
 
+/*
+** bool validateURL(std::string, int [optional, defaulted to 64])
+**
+** Input:
+**   strURL: A std::string URL to be be processed for validity.
+**   strErr: A std::string to be filled with any error messages.
+**   maxSize: An int to define the maximum size the URL can be;
+**            Optional, defaulting to 64.
+**
+** Return:
+**    boolean true|false result if the validation passes.
+**    strRrror: Filled with any error messages.
+*/
```

Comment:
Move this description comment block to the header file and format it according to the doxygen-compatible comment guidelines.

Reply:
complete.